config/locales/en.yml

```diff
@@ -27,6 +27,7 @@ en:
   parameter_required: "Parameter is required"
   missing_parameters: "Bad or missing parameters"
   unpermitted_parameters: "Unpermitted parameters"
+  unauthorized: "HTTP Token: Access denied"
```

Comment:
```suggestion
  unauthorized: "Bearer Token: Access denied"
```

Reply:
Thats the default thing that rails returns, I think its probably better to keep the message the same as it was before.